cumulusci/cli/cci.py

```diff
@@ -843,32 +867,63 @@ def service_connect():
 
 
 @service.command(name="info", help="Show the details of a connected service")
+@click.argument("service_type")
```

Comment:
Should we make this optional so you can get the info for the default service without specifying its name?